.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
```

Comment:
Do we want 2 months?  6?

Reply:
two works for me. lol this will probably mean that the issues get closed before the PR gets merged.